GETTING_STARTED.md

```diff
@@ -0,0 +1,415 @@
+# Getting started
+
+In this document you will find a step-by-step guide on how to get Eunomia running in a local Minikube cluster.
+
+Eunomia is watching GitOpsConfig Custom Resources. They and their handling by Eunomia will be discussed in general.
+
+You will run two examples:
+- Eunomia acting upon local GitOpsConfig changes,
+- Eunomia watching for changes in a remote repository and acting upon them.
+
+## Prerequisites
+
+In order to build Eunomia, you'll need the following:
+
+- [Git](https://git-scm.com/downloads)
+- [Go 1.12+](https://golang.org/dl/)
+- [Docker](https://docs.docker.com/install/)
+- [Operator SDK v0.8.1](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/doc/user/install-operator-sdk.md)
+- Access to a Kubernetes cluster
+  - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+  - [Minikube](https://kubernetes.io/docs/setup/minikube/) (for local development and testing)
+
+You will also have to [set up the GOPATH](https://github.com/golang/go/wiki/SettingGOPATH) env variable (the instructions are for Go 1.13 but it's the same for Go 1.12).
+
+### Installing on a Mac
+
+All the components (except operator-sdk v0.8.1) can be easily installed via [Homebrew](https://brew.sh/) on a Mac:
+
+```shell
+brew install git
+brew install go@1.12
+brew install docker
+brew install minikube
+brew install kubectl
+```
+
+Unfortunately, operator-sdk in version 0.8.1 isn't available via brew so you have to install it from GitHub release as described [on this page](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/doc/user/install-operator-sdk.md).
+
+### Checkout your fork
+
+You wil need your own Eunomia fork later in the second example. When pushing changes to your fork, you will observe Eunomia make adjustments based on your changed GitOpsConfig.
```

Comment:
1. `wil` &rarr; `will`
 2. Could you explicitly use the name of the example you're referring to instead of "the second example"? currently I'm not sure which one exactly is this about, where should I start counting.
 3. I'm kinda uneasy with asking people to fork whole eunomia repo. If we only need this for people to easily modify examples, then I don't like the first sentence here.  To me, it makes it sound kinda like a fork of *whole eunomia repo* is *necessary* for even *getting started* with using eunomia. 
I think I'd prefer it rephrased to make it clear eunomia can work with any custom scratch-built repo with kubernetes/helm yaml files, but just to make instructions shorter and easier to maintain for us, we'll be asking user to fork the eunomia repos, purely for the examples/ directory (if that's what will be used)